decidim-core/config/initializers/devise.rb

```diff
@@ -302,6 +302,33 @@ def i18n_options(options)
     manager.failure_app = RandomStalling
   end
 
+  # To be compatible with Turbo, from Devise v.4.9.0 on,
+  # devise keep error status for validation as :ok, and sets
+  # the redirect_status as :found. However, these configuration
+  # options is devised to change this behavior as
+  # needed (for more information refer to
+  # https://github.com/heartcombo/devise/blob/v4.9.0/CHANGELOG.md#490---2023-02-17):
+  config.responder.error_status = :forbidden
+
+  # In order to be compatible with the jwt authentication, we need to set these
+  # configurations.
+  # JWT secret is being used by the devise-jwt to generate the tokens, once the
+  # user authenticated.
+  config.jwt do |jwt|
+    jwt.secret = Rails.application.secrets.dig(:api, :secret_key_jwt)
+    next unless jwt.secret
+    next unless defined?(Decidim::Api)
```

Comment:
I think the whole `config.jwt` block should not be applied in case the `Api` module is not installed.

Or is there a reason to apply it regardless?

Reply:
This comment was not addressed.